images/ansible-operator/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ansible-runner = "==1.4.6"
+ansible-runner-http = "==1.0.0"
+ipaddress = "==1.0.23"
+openshift = "==0.11.2"
```

Comment:
I'd prefer if you left version changes to a follow-up PR, so issues with bumps are easily revertible. Unless there's a good reason to bump here.

Reply:
Yeah me too but openshift 0.10.3 contains a subdependency that contains a set of known vulnerabilities. You can check:
```
rein@rein-Lenovo-Y720-15IKB  ~  mkdir sec_check && cd sec_check       
 rein@rein-Lenovo-Y720-15IKB  ~/sec_check  pipenv --version
pipenv, version 2020.11.15
 rein@rein-Lenovo-Y720-15IKB  ~/sec_check  pipenv install openshift==0.10.3        
Creating a virtualenv for this project...
Pipfile: /home/rein/sec_check/Pipfile
Using /usr/bin/python3.8 (3.8.5) to create virtualenv...
⠼ Creating virtual environment...created virtual environment CPython3.8.5.final.0-64 in 326ms
  creator CPython3Posix(dest=/home/rein/.local/share/virtualenvs/sec_check-Mqnne5rg, clear=False, global=False)
  seeder FromAppData(download=False, distro=latest, lockfile=latest, CacheControl=latest, requests=latest, html5lib=latest, contextlib2=latest, packaging=latest, retrying=latest, ipaddr=latest, pyparsing=latest, pytoml=latest, msgpack=latest, chardet=latest, colorama=latest, six=latest, setuptools=latest, pip=latest, pkg_resources=latest, urllib3=latest, pep517=latest, wheel=latest, progress=latest, idna=latest, appdirs=latest, distlib=latest, certifi=latest, webencodings=latest, via=copy, app_data_dir=/home/rein/.local/share/virtualenv/seed-app-data/v1.0.1.debian)
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator

✔ Successfully created virtual environment! 
Virtualenv location: /home/rein/.local/share/virtualenvs/sec_check-Mqnne5rg
Installing openshift==0.10.3...
Adding openshift to Pipfile's [packages]...
✔ Installation Succeeded 
Installing dependencies from Pipfile.lock (95984d)...
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 6/6 — 00:00:00
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
 rein@rein-Lenovo-Y720-15IKB  ~/sec_check  
 ✘ rein@rein-Lenovo-Y720-15IKB  ~/sec_check  pipenv check                    
Checking PEP 508 requirements...
Passed!
Checking installed package safety...
36333: pyyaml <4 resolved (3.13 installed)!
Pyyaml before 4 uses ``yaml.load`` which has been assigned CVE-2017-18342.

38100: pyyaml <5.3.1 resolved (3.13 installed)!
A vulnerability was discovered in the PyYAML library in versions before 5.3.1, where it is susceptible to arbitrary code execution when it processes untrusted YAML files through the full_load method or with the FullLoader loader. Applications that use the library to process untrusted input may be vulnerable to this flaw. An attacker could use this flaw to execute arbitrary code on the system by abusing the python/object/new constructor. See: CVE-2020-1747.

39611: pyyaml <5.4 resolved (3.13 installed)!
A vulnerability was discovered in the PyYAML library in versions before 5.4, where it is susceptible to arbitrary code execution when it processes untrusted YAML files through the full_load method or with the FullLoader loader. Applications that use the library to process untrusted input may be vulnerable to this flaw. This flaw allows an attacker to execute arbitrary code on the system by abusing the python/object/new constructor. This flaw is due to an incomplete fix for CVE-2020-1747. See CVE-2020-14343.
```

Reply:
So openshift==0.10.3 won't pass the `pipenv check` phase in the base.Dockerfile

Reply:
Can you add a changelog entry stating that `openshift` has been bumped to 0.11.2?